packages/core/services/FileService/HttpFileService/index.ts

```diff
@@ -127,4 +129,29 @@ export default class HttpFileService extends HttpServiceBase implements FileServ
             uniqueId()
         );
     }
+
+    /**
+     * Cache a list of files to NAS cache (VAST) by sending their IDs to FSS.
+     */
+    public async cacheFiles(
+        fileIds: string[],
+        username?: string
+    ): Promise<{ cacheFileStatuses: { [fileId: string]: string } }> {
+        const requestUrl = `${this.aicsLoadBalancerBaseUrl}/${HttpFileService.BASE_FILE_CACHE_URL}${this.pathSuffix}`;
+        const requestBody = JSON.stringify({ fileIds });
+        const headers = {
+            "Content-Type": "application/json",
+            "X-User-Id": username || "anonymous",
+        };
+        console.log(headers);
```

Comment:
console.log

Reply:
resolved in [44b3f16](https://github.com/AllenInstitute/biofile-finder/pull/322/commits/44b3f168a061bf7397716f4d325e0b5a2fde6db0)